packages/ef-runtime-client/src/ui/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { render } from "preact/compat";
+import EFUI from "./efui";
+import { IComponentRegistry } from "../ComponentRegistry";
+
+function init(registry: IComponentRegistry) {
+  const efuiEl = document.createElement("div");
+  efuiEl.id = "efui";
+  document.body.appendChild(efuiEl);
+  render(<EFUI componentRegistry={registry} />, efuiEl);
+}
+
+export default { init };
```

Comment:
```suggestion
import { createPortal } from "preact/compat";

function init(registry: IComponentRegistry) {
  const efuiEl = document.createElement("div");
  efuiEl.id = "efui";
  document.body.appendChild(efuiEl);

  render(
    createPortal(<EFUI componentRegistry={registry} />, efuiEl),
    document.body
  );
}
```

This may be a better way of manipulating the dom if it works? I haven't had the opportunity to use create portal before but it may work here? Here are the [docs](https://react.dev/reference/react-dom/createPortal).

Reply:
I tried this. It doesn't work. I guess it might also be a webpack build issue

Reply:
Fair enough :) Worth a try!